akka-remote/src/main/scala/akka/remote/artery/ObjectPool.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+
+import org.agrona.concurrent.ManyToManyConcurrentArrayQueue
```

Comment:
I'm sure we will notice if they remove the dependency. Our code will not compile. We might not notice until runtime (after release) if we are using the wrong version.

That said, I can reconsider when we release a final, which will also rely on final version of Aeron.